examples/e-commerce/src/theme.css

```diff
@@ -10,6 +10,7 @@ a[class^='ais-'] {
   text-decoration: none;
 }
 
+[class^='ais-'][class$='--disabled'],
```

Comment:
This adds compatibility to Angular components.

Reply:
Since the goal is to have the same DOM output everywhere (eventually), shouldn't we add here a comment reminding to remove this when no longer necessary?

Reply:
WDYM?

As I understand it, it's needed for Angular because it creates web components. Those are not targetable with the `*` selector.